docs/user-guide/cli-ftpplugin.md

```diff
@@ -13,6 +13,10 @@ As a z/OS user, you can use the plug-in to perform the following tasks:
 
 ## Commands
 
+:::note
+When issuing commands, use only ASCII characters. FTP does not support non-ASCII characters, such as glyphs or mathematical symbols.
```

Comment:
The issue in this case is not that the command does not contain ASCII characters. The issue in this case is that the file/dataset/dataset member being transferred contained characters that are not contained in the ISO-8859-1 (Latin-1/Extended ASCII) character set. When this occurs, a translation error character takes the place of the character that could not be represented, and if uploaded back to the mainframe, will result in a translation error and data loss.

Reply:
thank you! updated based on your comments